### ISSUE TITLE
`{:parse {:interpose "\n\n"}}` causes newlines to be removed after the last form

### ISSUE THREAD

**Issue by mynomoto** [kkinnear/zprint#100]
```
$ zprint '{:parse {:interpose "\n\n"}}' <<< $'(ns foo)(defn baz [])'
(ns foo)

(defn baz [])
```
Is the same as:
```
$ zprint '{:parse {:interpose "\n\n"}}' <<< $'(ns foo)(defn baz [])\n\n'
(ns foo)

(defn baz [])
```
Could this be configured or at least preserved?
Thanks!

**Comment by kkinnear** [comment]
The following is also true:
```clojure
$ ./zprintm-0.4.15  <<< $'(ns foo)\n\n(defn baz [])\n\n'
(ns foo)

(defn baz [])
$ ./zprintm-0.4.15  <<< $'(ns foo)\n\n(defn baz [])'
(ns foo)

(defn baz [])
$
```
I believe that you are asking for the "\n\n" at the end to be preserved.  Do you want it preserved when you use `{:parse {:interpose "\n\n"}}` or normally (i.e., without `:parse`), or in both cases?

Obviously right now any extra blank lines at the end go away.  Preserving them normally and preserving them when using  `{:parse {:interpose "\n\n"}}` are likely to be different tasks (if memory serves me correctly).  So, if I'm going to do what you want (which I would like to do and think I can do), it would be nice to know in a bit more detail precisely what you want.

Thanks!

**Comment by mynomoto** [comment]
Actually, without the interpose there is a extra new line at your example, trying to show the difference below:
```clojure
; with interpose
$ zprintl-0.4.5 '{:parse {:interpose "\n\n"}}' <<< $'(ns foo)\n\n(defn baz [])'
(ns foo)

(defn baz
  [])
$

; without interpose
$ zprintl-0.4.5 <<< $'(ns foo)\n\n(defn baz [])'
(ns foo)

(defn baz
  [])

$
```
I only noticed that on a diff on github when it pointed me files that ended without the new-line char. I also tested and it makes no difference how many new line chars there are in the end, with `{:parse {:interpose "\n\n"}}` it always retuns with none and without it returns with a single `\n`.

I would like that it always returned with a single `\n` in the end, or that it could be configured. Thanks!

**Comment by kkinnear** [comment]
I'm sure glad I asked!  I'll work toward ensuring there is a `\n` when interpose used, so that they are the same.  That seems like something safe enough.  Making it configurable seems a bit much even for me, who obviously thinks just about everything should be configurable.  Thanks.

**Comment by kkinnear** [comment]
I've fixed this in version 0.4.16.  Finally!  I hope that it works consistently now.
